php/php.code.analysis/src/org/netbeans/modules/php/analysis/options/AnalysisOptionsValidator.java

```diff
@@ -73,7 +73,7 @@ public AnalysisOptionsValidator validateCodeSnifferPath(String codeSnifferPath)
 
     @NbBundle.Messages("AnalysisOptionsValidator.codeSniffer.standard.empty=Valid code sniffer standard must be set.")
     public AnalysisOptionsValidator validateCodeSnifferStandard(String codeSnifferStandard) {
```

Comment:
I'm not sure about keeping or removing this method.

Reply:
You cannot remove it since it is still used in the file below. Try to remove it and compile the module.

Reply:
Yes, now its used at more places.
Pesonally I would removed this method and all its usages, because I don't see any value testing if standard is not null as in other places null value is replaced by empty string.

Reply:
Yes, now its used at more places.
Personally I would removed this method and all its usages, because I don't see any value testing if standard is not null. There are places where null is replaced by empty string.

Reply:
If it worked, I would not be against such a change, of course :)

Reply:
Maybe @junichi11 could also help here.

Reply:
Here: https://github.com/apache/netbeans/blob/ac4e72f120fc01463c525f357d25c43c81e34a6f/php/php.code.analysis/src/org/netbeans/modules/php/analysis/options/AnalysisOptions.java#L168

Reply:
Well, maybe we can't remove this. It can be `null`.

See 
https://github.com/apache/netbeans/blob/2e7a2e98a2908dbed3897565621073ef263b286f/php/php.code.analysis/src/org/netbeans/modules/php/analysis/options/AnalysisOptionsValidator.java#L38-L42

https://github.com/apache/netbeans/blob/ac4e72f120fc01463c525f357d25c43c81e34a6f/php/php.code.analysis/src/org/netbeans/modules/php/analysis/ui/options/CodeSnifferOptionsPanel.java#L187

https://github.com/apache/netbeans/blob/ac4e72f120fc01463c525f357d25c43c81e34a6f/php/php.code.analysis/src/org/netbeans/modules/php/analysis/ui/options/CodeSnifferOptionsPanel.java#L121-L127

Reply:
OK, I'll leave the method as it is.